.github/workflows/release.yml

```diff
@@ -18,7 +19,7 @@ jobs:
           git config --global user.email 'github@scaleway.com'
           git remote set-url origin https://x-access-token:${GITHUB_TOKEN}@github.com/$GITHUB_REPOSITORY
         env:
-          GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Why ? These are not the same, GH_TOKEN is our token 😄

Reply:
@chambo-e why not using injected one by github?

Reply:
It's a scalway-bot account PAT, so we'd release with it

But that does not seems to have any effect now so 🤷